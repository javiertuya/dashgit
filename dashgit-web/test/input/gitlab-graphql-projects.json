{
    "data": {
        "projects": {
            "nodes": [
                {
                    "id": "gid://gitlab/Project/4", "archived": true, "fullPath": "org/proj4", "lastActivityAt": "2024-02-04T14:49:00+01:00",
                    "webUrl": "https://mygitlab.com/org/proj4",
                    "repository": { "branchNames": [ "main", "feature" ] }
                },
                {
                    "id": "gid://gitlab/Project/5", "archived": false, "fullPath": "org1/proj5", "lastActivityAt": "2024-02-05T15:50:00+01:00",
                    "webUrl": "https://mygitlab.com/org1/proj5",
                    "repository": { "branchNames": [ "feature", "main", "building" ] }
                },
                {
                    "id": "gid://gitlab/Project/6", "archived": false, "fullPath": "org2/proj6", "lastActivityAt": "2024-02-06T16:49:00+01:00",
                    "webUrl": "https://mygitlab.com/org2/proj6",
                    "repository": { "branchNames": [ "main"] }
                },
                {
                    "id": "gid://gitlab/Project/7", "archived": false, "fullPath": "org2/proj7", "lastActivityAt": "2024-02-07T17:49:00+01:00",
                    "webUrl": "https://mygitlab.com/org2/proj7",
                    "repository": { "branchNames": [ "branch1"] }
                }
            ]
        }
    }
}