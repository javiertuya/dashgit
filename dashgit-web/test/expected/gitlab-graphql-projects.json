{
  "header": {
    "provider": "GitLab",
    "uid": "0-gitlab",
    "user": "usr1",
    "url": "https://mygitlab.com",
    "repo_names": [
      "org1/proj5",
      "org2/proj6",
      "org2/proj7"
    ],
    "target": "",
    "message": ""
  },
  "items": [
    {
      "repo_name": "org1/proj5",
      "type": "branch",
      "iid": "",
      "branch_name": "feature",
      "status": "notavailable",
      "title": "",
      "actions": {},
      "author": "",
      "assignees": "",
      "created_at": "",
      "updated_at": "",
      "iidstr": "",
      "url": "",
      "branch_url": "",
      "repo_url": "https://mygitlab.com/org1/proj5",
      "labels": [],
      "gid": "gid://gitlab/Project/5",
      "uid": "org1-proj5_branch_feature"
    },
    {
      "repo_name": "org1/proj5",
      "type": "branch",
      "iid": "",
      "branch_name": "main",
      "status": "notavailable",
      "title": "",
      "actions": {},
      "author": "",
      "assignees": "",
      "created_at": "",
      "updated_at": "",
      "iidstr": "",
      "url": "",
      "branch_url": "",
      "repo_url": "https://mygitlab.com/org1/proj5",
      "labels": [],
      "gid": "gid://gitlab/Project/5",
      "uid": "org1-proj5_branch_main"
    },
    {
      "repo_name": "org1/proj5",
      "type": "branch",
      "iid": "",
      "branch_name": "building",
      "status": "notavailable",
      "title": "",
      "actions": {},
      "author": "",
      "assignees": "",
      "created_at": "",
      "updated_at": "",
      "iidstr": "",
      "url": "",
      "branch_url": "",
      "repo_url": "https://mygitlab.com/org1/proj5",
      "labels": [],
      "gid": "gid://gitlab/Project/5",
      "uid": "org1-proj5_branch_building"
    },
    {
      "repo_name": "org2/proj6",
      "type": "branch",
      "iid": "",
      "branch_name": "main",
      "status": "notavailable",
      "title": "",
      "actions": {},
      "author": "",
      "assignees": "",
      "created_at": "",
      "updated_at": "",
      "iidstr": "",
      "url": "",
      "branch_url": "",
      "repo_url": "https://mygitlab.com/org2/proj6",
      "labels": [],
      "gid": "gid://gitlab/Project/6",
      "uid": "org2-proj6_branch_main"
    },
    {
      "repo_name": "org2/proj7",
      "type": "branch",
      "iid": "",
      "branch_name": "branch1",
      "status": "notavailable",
      "title": "",
      "actions": {},
      "author": "",
      "assignees": "",
      "created_at": "",
      "updated_at": "",
      "iidstr": "",
      "url": "",
      "branch_url": "",
      "repo_url": "https://mygitlab.com/org2/proj7",
      "labels": [],
      "gid": "gid://gitlab/Project/7",
      "uid": "org2-proj7_branch_branch1"
    }
  ]
}